### PR TITLE
Stop checking if user exists in WIT during login

### PR DESCRIPTION
**Pull request by alexeykazakov**
We don't need to check the user in WIT during login anymore. Auth DB is now in sync with WIT.
This part of the cleanup phase from - https://github.com/fabric8-services/fabric8-auth/issues/79

### CODE REVIEW

**Comment by fabric8cd** [comment]
@alexeykazakov snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-119-1`

**Comment by fabric8cd** [comment]
@alexeykazakov snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-119-2`

**Comment by alexeykazakov** [comment]
[test]

**Comment by fabric8cd** [comment]
@alexeykazakov snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-119-3`

**Comment by alexeykazakov** [comment]
[test]

**Comment by alexeykazakov** [comment]
[test]

**Comment by fabric8cd** [comment]
@alexeykazakov snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-119-4`

File: login/service.go [hunk 1]
```diff
@@ -67,7 +65,7 @@ type KeycloakOAuthProvider struct {
 // KeycloakOAuthService represents keycloak OAuth service interface
 type KeycloakOAuthService interface {
 	Perform(ctx *app.LoginLoginContext, config *oauth2.Config, serviceConfig LoginServiceConfiguration) error
-	CreateOrUpdateKeycloakUser(accessToken string, ctx context.Context, profileEndpoint string, user *account.User, identity *account.Identity) (*account.User, *account.Identity, error)
+	CreateOrUpdateIdentity(accessToken string, ctx context.Context, profileEndpoint string) (*account.Identity, bool, error)
```
Comment (sbose78) [inline login/service.go]: Good cleanup, returning both identity & user objects were indeed redundant.

**Comment by sbose78** [comment]
[test]

**Comment by alexeykazakov** [comment]
[test]

**Pull request merged**